client/src/Component/views/Play/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Header from '../../Common/Header';
+import Footer from '../../Common/Footer';
+
+const Play = () => {
+  return (
+    <div>
+      <Header titleText="Choose your game play:" />
```

Comment:
There seems to be an issue with the component not being supplied an image src:
![Screen Shot 2019-04-19 at 3 15 48 PM](https://user-images.githubusercontent.com/9094166/56428150-46a06f00-62b6-11e9-8647-141267493c43.png)

Reply:
I think if you include the prop with a type of null like Ali suggested it should fix the errors!
``` <Header titleText="Choose your game play:" headerImg={null} />```

Reply:
you are totally right :)